src/sentry/utils/snuba.py

```diff
@@ -204,6 +204,7 @@ def increment(
         if error and isinstance(error, urllib3.exceptions.ReadTimeoutError):
             raise six.reraise(type(error), error, _stacktrace)
 
+        metrics.incr("snuba.client.retry", sample_rate=1.0, tags={"method": method, "url": url})
```

Comment:
This sample rate might be a bit higher than necessary, @JTCunning may have an opinion. I'm also don't recall us passing any query string that could cause this URL to be high cardinality — if that's not true, we should probably not include it as a tag.

Reply:
Yeah, I'm not really sure what the considerations are with `sample_rate`, I assume just cost?

Reply:
I would start at the default (10%) and increase it out of necessity instead of opportunity.

While I'm here, I would also suggest that we do not use the full URL as a tag, and just the location.